flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/stream/table/OverWindowTest.scala

```diff
@@ -53,20 +54,20 @@ class OverWindowTest extends TableTestBase {
           unaryNode(
             "DataStreamCalc",
             streamTableNode(0),
-            term("select", "a", "b", "c", "proctime")
+            term("select", "a", "b", "c", "proctime", "1 AS $4")
```

Comment:
This looks like a bug to me. Why should the constant be pushed down? Could you elaborate this case?

Reply:
due to "[CALCITE-2726] RexSimplify does not simplify enough for some of the nested operations, doesn't do nested visitCall". this looks like something we have to fix on Calcite side. do you think we can proceed with this test change and later fix it?